exporter/jaeger/opentelemetry-exporter-jaeger.gemspec

```diff
@@ -27,7 +27,7 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '>= 3.0'
 
   spec.add_dependency 'opentelemetry-api', '~> 1.1'
-  spec.add_dependency 'opentelemetry-common', '~> 0.19.6'
+  spec.add_dependency 'opentelemetry-common', '~> 0.20'
```

Comment:
I loosened the constrain here and bumped the minimum required version, not sure if anyone has any concerns on that. That was what we did in the past before contrib, I think it should still be ok now.

Reply:
Works for me. I play this game with contrib all the time.